variables.tf

```
@@ -57,9 +57,31 @@ variable "github_admins" {
   description = "A list of Github teams that should have admins access"
 }
 
-variable "github_organization" {
-  type        = string
-  description = "The Github organization to connect the workspace to"
+variable "github_branch_protection" {
```

```suggestion
variable "branch_protection" {
```

Just to keep it inline with the rest and a bit shorter... Sorry 😊